[2025-09-17T01:58:04.961+0000] {processor.py:161} INFO - Started process (PID=188) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T01:58:04.967+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T01:58:04.970+0000] {logging_mixin.py:188} INFO - [2025-09-17T01:58:04.969+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T01:58:04.989+0000] {logging_mixin.py:188} INFO - [2025-09-17T01:58:04.981+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T01:58:04.992+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T01:58:05.083+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.154 seconds
[2025-09-17T01:58:35.912+0000] {processor.py:161} INFO - Started process (PID=226) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T01:58:35.914+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T01:58:35.916+0000] {logging_mixin.py:188} INFO - [2025-09-17T01:58:35.916+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T01:58:35.926+0000] {logging_mixin.py:188} INFO - [2025-09-17T01:58:35.924+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T01:58:35.927+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T01:58:35.999+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.094 seconds
[2025-09-17T01:59:07.640+0000] {processor.py:161} INFO - Started process (PID=254) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T01:59:07.646+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T01:59:07.652+0000] {logging_mixin.py:188} INFO - [2025-09-17T01:59:07.650+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T01:59:07.672+0000] {logging_mixin.py:188} INFO - [2025-09-17T01:59:07.670+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T01:59:07.674+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T01:59:07.787+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.156 seconds
[2025-09-17T01:59:38.930+0000] {processor.py:161} INFO - Started process (PID=279) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T01:59:38.936+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T01:59:38.941+0000] {logging_mixin.py:188} INFO - [2025-09-17T01:59:38.940+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T01:59:38.965+0000] {logging_mixin.py:188} INFO - [2025-09-17T01:59:38.959+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T01:59:38.967+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T01:59:39.088+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.169 seconds
[2025-09-17T02:00:09.792+0000] {processor.py:161} INFO - Started process (PID=304) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:00:09.799+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T02:00:09.802+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:00:09.801+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:00:09.830+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:00:09.825+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T02:00:09.833+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:00:09.946+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.165 seconds
[2025-09-17T02:04:09.167+0000] {processor.py:161} INFO - Started process (PID=187) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:04:09.168+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T02:04:09.171+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:04:09.170+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:04:09.189+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:04:09.186+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T02:04:09.192+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:04:09.272+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.111 seconds
[2025-09-17T02:04:41.310+0000] {processor.py:161} INFO - Started process (PID=226) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:04:41.313+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T02:04:41.316+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:04:41.314+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:04:41.398+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:04:41.381+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T02:04:41.413+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:04:41.818+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.519 seconds
[2025-09-17T02:05:12.481+0000] {processor.py:161} INFO - Started process (PID=252) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:05:12.490+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T02:05:12.521+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:05:12.512+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:05:12.632+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:05:12.590+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T02:05:12.648+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:05:13.175+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.713 seconds
[2025-09-17T02:05:43.555+0000] {processor.py:161} INFO - Started process (PID=277) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:05:43.557+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T02:05:43.559+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:05:43.559+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:05:43.572+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:05:43.571+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T02:05:43.573+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:05:43.604+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.053 seconds
[2025-09-17T02:06:14.554+0000] {processor.py:161} INFO - Started process (PID=302) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:06:14.557+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T02:06:14.562+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:06:14.561+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:06:14.578+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:06:14.577+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T02:06:14.579+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:06:14.614+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.069 seconds
[2025-09-17T02:06:45.172+0000] {processor.py:161} INFO - Started process (PID=327) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:06:45.176+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T02:06:45.182+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:06:45.181+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:06:45.198+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:06:45.192+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T02:06:45.200+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:06:45.277+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.121 seconds
[2025-09-17T02:07:16.175+0000] {processor.py:161} INFO - Started process (PID=352) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:07:16.178+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T02:07:16.181+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:07:16.180+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:07:16.207+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:07:16.201+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T02:07:16.209+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:07:16.331+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.166 seconds
[2025-09-17T02:07:46.930+0000] {processor.py:161} INFO - Started process (PID=387) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:07:46.932+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T02:07:46.933+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:07:46.933+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:07:46.976+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:07:46.968+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T02:07:46.977+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:07:47.104+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.180 seconds
[2025-09-17T02:08:17.887+0000] {processor.py:161} INFO - Started process (PID=412) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:08:17.890+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T02:08:17.893+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:08:17.892+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:08:17.904+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:08:17.901+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T02:08:17.905+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:08:17.963+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.083 seconds
[2025-09-17T02:08:53.093+0000] {processor.py:161} INFO - Started process (PID=437) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:08:53.097+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T02:08:53.101+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:08:53.100+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:08:53.145+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:08:53.140+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T02:08:53.147+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:08:53.301+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.259 seconds
[2025-09-17T02:09:23.820+0000] {processor.py:161} INFO - Started process (PID=464) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:09:23.825+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T02:09:23.829+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:09:23.828+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:09:23.841+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:09:23.838+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T02:09:23.842+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:09:24.011+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.201 seconds
[2025-09-17T02:09:55.105+0000] {processor.py:161} INFO - Started process (PID=490) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:09:55.110+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T02:09:55.112+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:09:55.111+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:09:55.128+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:09:55.126+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T02:09:55.128+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:09:55.164+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.068 seconds
[2025-09-17T02:10:26.046+0000] {processor.py:161} INFO - Started process (PID=538) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:10:26.062+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T02:10:26.072+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:10:26.069+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:10:26.111+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:10:26.108+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T02:10:26.112+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:10:26.710+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.703 seconds
[2025-09-17T02:10:57.393+0000] {processor.py:161} INFO - Started process (PID=572) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:10:57.396+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T02:10:57.398+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:10:57.397+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:10:57.412+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:10:57.409+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T02:10:57.412+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:10:57.509+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.131 seconds
[2025-09-17T02:11:40.472+0000] {processor.py:161} INFO - Started process (PID=201) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:11:40.475+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T02:11:40.477+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:11:40.476+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:11:40.508+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:11:40.503+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T02:11:40.510+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:11:40.749+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.285 seconds
[2025-09-17T02:12:10.894+0000] {processor.py:161} INFO - Started process (PID=227) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:12:10.896+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T02:12:10.898+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:12:10.897+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:12:10.912+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:12:10.910+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T02:12:10.913+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:12:11.007+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.122 seconds
[2025-09-17T02:12:43.335+0000] {processor.py:161} INFO - Started process (PID=254) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:12:43.341+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T02:12:43.353+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:12:43.349+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:12:43.424+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:12:43.419+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T02:12:43.426+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:12:43.542+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.219 seconds
[2025-09-17T02:13:14.891+0000] {processor.py:161} INFO - Started process (PID=279) to work on /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:13:14.896+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/full_pipeline_test_dag.py for tasks to queue
[2025-09-17T02:13:14.899+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:13:14.898+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:13:14.918+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:13:14.916+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/full_pipeline_test_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/full_pipeline_test_dag.py", line 6, in <module>
    from scripts.crypto_producer import run_crypto_tick_producer
ModuleNotFoundError: No module named 'scripts.crypto_producer'
[2025-09-17T02:13:14.919+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/full_pipeline_test_dag.py
[2025-09-17T02:13:15.045+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/full_pipeline_test_dag.py took 0.191 seconds
